pocket_lab/pl_gui_iio_wrapper.c

```diff
@@ -270,12 +270,12 @@ int32_t pl_gui_scan_global_attr_avail_options(const char *attr_name,
  * @param	dev_indx[in] - Current device index
  * @return	0 in case of success, negative error code otherwise
  */
-int32_t pl_gui_scan_chn_attr_avail_options(const char *attr_name,
+int32_t pl_gui_get_chn_attr_avail_options(const char *attr_name,
 		char *attr_val, uint32_t chn_indx, uint32_t dev_indx)
 {
 	struct iio_device *iio_dev;
 	struct iio_ch_info chn_info;
-	char buf[50];
+	char buf[100];
```

Comment:
Was this change intentional?

Reply:
Nvm, it probably makes sense to have a larger buffer to accommodate all the options for an attr.

Reply:
ya, that was the reason the size is increased